tests/test_data_track.py

```
@@ -308,15 +328,17 @@
 
     rows = (
         ctx.db.get_fingerprint_db()
-        .execute("SELECT id, track_id FROM fingerprint ORDER BY id")
+        .execute(text("SELECT id, track_id FROM fingerprint ORDER BY id"))
         .fetchall()
     )
     assert [(1, 3), (2, 3)] == rows
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `rows == [(1, 3), (2, 3)]`

(SIM300)